parsec/api/data/sequester.py

```diff
@@ -0,0 +1,39 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+
+from enum import Enum
+from typing import Dict, Any
+import attr
+from parsec.api.data.base import BaseAPIData
+from parsec.serde import fields, post_load
+from parsec.serde.schema import BaseSchema
+from pendulum import DateTime
+
+
+class EncryptionKeyFormat(Enum):
+    RSA = "RSA"
+
+
+EncryptionKeyFormatFields = fields.enum_field_factory(EncryptionKeyFormat)
+
+
+@attr.s(slots=True, frozen=True, auto_attribs=True, kw_only=True, eq=False)
+class SequesterServiceEncryptionKey(BaseAPIData):
+    class SCHEMA_CLS(BaseSchema):
+
+        type = fields.CheckedConstant("sequester_service_encryption_key", required=True)
```

Comment:
If we rename the class to `SequesterServiceCertificate`, can you add a comment "No author field here given we are signed by the sequester authority"